sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/implementation/oauth2/AbstractOAuth2AuthorizationCodeGrantRequestEntityConverter.java

```diff
@@ -28,22 +36,15 @@
     protected abstract String getApplicationId();
 
     @Override
-    @SuppressWarnings("unchecked")
     public RequestEntity<?> convert(OAuth2AuthorizationCodeGrantRequest request) {
-        addHeadersConverter(headersConverter);
-        addParametersConverter(parametersConverter);
         return super.convert(request);
     }
 
-    private final Converter<OAuth2AuthorizationCodeGrantRequest, HttpHeaders> headersConverter = (request) -> getHttpHeaders();
-
-    private final Converter<OAuth2AuthorizationCodeGrantRequest, MultiValueMap<String, String>> parametersConverter = this::getHttpBody;
-
     /**
      * Additional default headers information.
      * @return HttpHeaders
      */
-    public HttpHeaders getHttpHeaders() {
+    public HttpHeaders getHttpHeaders(OAuth2AuthorizationCodeGrantRequest request) {
```

Comment:
Why changing this?

Reply:
Just accept suggestion from here: https://github.com/Azure/azure-sdk-for-java/pull/31213/files#r990644464
![image](https://user-images.githubusercontent.com/13167207/195746656-645b014c-33da-4b7d-861c-cf32cedba586.png)

Reply:
Should `getHttpHeaders` and `getHttpBody` be `protected` then?